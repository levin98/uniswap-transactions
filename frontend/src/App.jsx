import React from 'react';
import TransactionList from './components/TransactionList';
import './App.css';

function App() {
  return (
    <div className="App">
      <TransactionList />
    </div>
  );
}

export default App;
