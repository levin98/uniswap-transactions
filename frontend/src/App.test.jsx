import React from 'react';
import { render, screen } from '@testing-library/react';
import App from './App';
import "./mock"

// eslint-disable-next-line react/display-name
jest.mock('./components/TransactionList', () => () => <div data-testid="transaction-list">Mocked Transaction List</div>);

test('renders App with Transaction List', () => {
  const { container } = render(<App />);
  expect(screen.getByTestId("transaction-list")).toBeInTheDocument();
  expect(container).toMatchSnapshot();
});
