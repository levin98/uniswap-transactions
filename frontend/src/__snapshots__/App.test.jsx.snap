// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App with Transaction List 1`] = `
<div>
  <div
    class="App"
  >
    <div
      data-testid="transaction-list"
    >
      Mocked Transaction List
    </div>
  </div>
</div>
`;
